File: 1.rb
sum = 0
1000.times do |i|
  if i % 3 == 0 or i % 5 ==0
    sum = sum + i
  end
end

puts sum

puts (1..1000).to_a.select {|x| x%3==0 or x%5==0}.inject(0){|b,i| b+i}

    

File: golang/euler/1.go
package main

import "fmt"

func main(){
    i := 1;
    sum := 0
    for i < 1000{
        if i%3==0 || i%5==0 {
            sum += i;
        }
        i += 1;
    }
    fmt.Println(sum)
}
